packages/cookie-consent-react/src/CookieConsent.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect } from "react";
+import { usePreviousValue } from "@fremtind/jkl-react-hooks";
+import { Consent } from "./types";
+import { useCookieConsentState } from "./CookieConsentContext";
+import { CookieConsentModal } from "./CookieConsentModal";
+import { setConsentCookie } from "./cookieConsentUtils";
+
+interface Props {
+    blocking?: boolean;
+    onAccept?: (consent: Consent) => void;
+}
```

Comment:
Ville nok som sagt hentet ut `onAccept` fra `CookieConsentModal`:
```suggestion
type ModalProps = React.ComponentProps<typeof CookieConsentModal>;

interface Props extends Partial<ModalProps> {
    blocking?: boolean;
}
```

eller hentet fra `types`:
```suggestion
interface Props extends ConsentComponentBaseProps {
    blocking?: boolean;
}
```

Reply:
Jeg definerte den som en egen type som begge interfacene kan ta inn. Grunnen til dette er at typen eksponert for brukeren i CookieConsent egentlig ikke har noe med den interne typen `ConsentComponentBaseProps` som kun ligger på de interne komponentene og aldri skal synes for en konsument.